.github/workflows/functional_tests.yml

```diff
@@ -0,0 +1,25 @@
+name: Functional tests using Ragger
+
+on:
+  workflow_dispatch:
+  push:
+    branches:
+      - master
+      - main
+      - develop
+  pull_request:
+
+jobs:
+  build_appplication:
+    name: Build application using the reusable workflow
+    uses: LedgerHQ/ledger-app-workflows/.github/workflows/build.yml@master
+    with:
+      uploaded_artifact_name: compiled_app_binaries
+
+  functionnal_tests:
```

Comment:
Don't you want to call this `ragger_tests` to advertise a bit more our framework ; ) ? 

We can still explain in the name what it does : `Run functional tests using ragger through reusable workflow`

Reply:
Done